src/shared/containers/DashboardResultsContainer.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useNexus } from '@bbp/react-nexus';
+import { Spin, Alert } from 'antd';
+import ResulTable from '../components/ResultsTable/ResultsTable';
+
+export type Binding = {
+  [key: string]: {
+    dataType?: string;
+    type: string;
+    value: string;
+  };
+};
+
+type Item = {
+  id: string;
+  self: string;
+  key: string;
+  [key: string]: any;
+};
+
+const DashboardResultsContainer: React.FunctionComponent<{
+  dataQuery: string;
+  orgLabel: string;
+  projectLabel: string;
+  viewId: string;
+  handleClick: (self: string) => void;
+}> = props => {
+  const { loading, data, error } = useNexus<any>(
+    nexus =>
+      nexus.View.sparqlQuery(
+        props.orgLabel,
+        props.projectLabel,
+        encodeURIComponent(props.viewId),
+        props.dataQuery
+      ),
+    [props.dataQuery, props.viewId] // only trigger new call if we have a new query and a new view
+  );
+
+  if (error) {
+    return (
+      <Alert
+        message="Error loading dashboard"
+        description={`Something went wrong: ${error.message || error}`}
+        type="error"
+      />
+    );
+  }
+
+  // build header properties
+  const headerProperties: {
+    title: string;
+    dataIndex: string;
+  }[] =
+    data &&
+    data.head.vars
+      .filter(
+        // we don't want to display total or self url in result table
```

Comment:
This is copied over from studio prototype. I am not sure if this logic makes sense here. ie, Will always have total, self etc in the result?

Reply:
Yeh, I think we are going to document this. `self` and `total` are normally used for creating links and pagination respectively. We normally don't need to display them in the Data table.